jscomp/Makefile

```diff
@@ -10,10 +10,10 @@ BYTE=ocamlc.opt$(EXE)
 OCAMLLEX=ocamllex.opt$(EXE)
 CAMLP4OF=camlp4of
 CAMLDEP=ocamldep.opt$(EXE)
-COMPFLAGS= -g -w +6-40-30 -warn-error +a-40-30
+COMPFLAGS= -g -w +6-40-30 -warn-error +a-40-30-23
 
 
-.SUFFIXES: .mli .ml .cmi .cmx .mll
+.SUFFIXES: .mli .ml .cmi .cmx .mll .c .o
```

Comment:
note, `make` by default will add `.c` to suffixes, with `-r` this will be exposed